install/kubernetes/cilium/values.yaml

```diff
@@ -14,6 +14,9 @@ rbac:
   # -- Enable creation of Resource-Based Access Control configuration.
   create: true
 
+# -- Default container images registry
+registry: quay.io/cilium
```

Comment:
One possible issue here - this is technically not a registry. `quay.io` is a registry, `cilium/operator` is an image name. We could argue that `cilium` in this case would be an organization, but I have mixed feelings about further splitting these components into separate variables.